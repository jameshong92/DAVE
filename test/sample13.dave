// test for general array
int main() {
    int[] num = {7, 8, 9};
    int sum = 0;
    size = sizeof(num)/sizeof(int);
    for (int i=0; i<size; i++) {
        print(num[i]);
        sum += num[i];
    }
    print(sum);
}
