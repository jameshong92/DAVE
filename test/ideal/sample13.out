7
8
9
24
